### ISSUE TITLE
Release a version for 2.0.0

### ISSUE THREAD

**Issue by LeoColomb** [roots/acorn#89]
Thanks for the impressive work for v2.0.0! 👍 

A nice little hint if it would be possible to release a new version for v2.0.0, or maybe a v2.0.0-beta or -rc if you're not confident enough for being production ready.

Thanks! ❤️

**Comment by QWp6t** [comment]
We'll tag the first beta on Monday. 🚀

**Comment by LeoColomb** [comment]
Thanks for the update @QWp6t! 🙂 

---
That said, if I may, it's not the first time I'm confused by Roots' open-source community management.
For exemple here, if resolution is on Monday, why did you close the ticket now?
Also why not releasing a version before? Any specific reasons?

I really appreciate your work but really sometimes it's very hard to follow...

**Comment by stefanfisk** [comment]
I might be missing something, but I cannot find any beta tag or release?

**Comment by str** [comment]
I can't find any documentation for how to use acorn.  Is there going to be documentation for v2.0?
